rpc/jsonrpc/types/chainsvrresults.go

```diff
@@ -26,14 +26,6 @@ type DecodeScriptResult struct {
 	P2sh      string   `json:"p2sh,omitempty"`
 }
 
-// EstimateSmartFeeResult models the data returned from the estimatesmartfee
-// command.
-type EstimateSmartFeeResult struct {
-	FeeRate float64  `json:"feerate"`
-	Errors  []string `json:"errors"`
-	Blocks  int64    `json:"blocks"`
-}
-
```

Comment:
This is in a standalone package and is exported, so I don't think it should be removed without a version update

Reply:
I see. I'm guessing it was used when estimatesmartfee had other modes. Can mark as depricated or should just leave it for now? What is the procedure with version updates?

Reply:
You can just add to the comment:

```Go
//
// Deprecated:  This will be removed in the next major version bump.
// Use the type exported from the JSON-RPC types module instead.
```